frontend/src/app/api.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import {HttpClient, HttpHeaders} from "@angular/common/http";
+import {Observable} from "rxjs";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+
+  baseurl = "http://127.0.0.1:8000/";
+  httpHeaders = new HttpHeaders({'Content-type': 'application/json'})
+
+  constructor(private http: HttpClient) { }
+
+  getAllChapters(): Observable<any>{
```

Comment:
I'd recommend to avoid `<any>` whereever it's possible. Going this way you loose all benefits of TypeScript.